packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -500,6 +515,31 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
    * @public
    */
   public canvasToWorld = (canvasPos: Point2): Point3 => {
+    const vtkCamera = this.getVtkActiveCamera() as vtkSlabCameraType;
+
+    /**
+     * NOTE: this is necessary because we want the coordinate trasformation
+     * respect to the view plane (plane orthogonal to the camera and passing to
+     * the focal point).
+     *
+     * When vtk.js computes the coordinate transformations, it simply uses the
+     * camera matrix (no ray casting).
+     *
+     * However for the volume viewport the clipping range is set to be
+     * (0.01, radius * 2), where the radius is radius containing all the actors
+     * in the viewport. This means that vkt.js will not return the coordinates
+     * of the point on the view plane
+     * (i.e. the depth coordinate will corresponde to a distance 0.01 from the
+     * camera position).
+     *
+     * Therefore the clipping range has to be set to (distance, distance + 0.01),
+     * where now distance is the distance between the camera position and focal
+     * point. This is done internally, in our camera customization when the flag
+     * isPerformingCoordinateTransformation is set to true.
+     */
```

Comment:
Nice explanation, can we add this expalnation as a comment to the vtkSlabCamera where we are using the `isPerforming....`?